client/src/services/index.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const registerUser = async () => {
+  const url = 'http://localhost:8080/';
```

Comment:
don't hardcode URL's like this. This url should be defined in a `.env` and then you add paths in your service layer like:

```js
const { API_BASE_URL } = process.env;

const registerUser = (body) => axios.post(`${API_BASE_URL}/users`, body);
const getUser = (id) => axios.get(`${API_BASE_URL}/users/${id}`);
```